internal/environment/check_env_flag_test.go

```diff
@@ -19,18 +19,17 @@
 package environment
 
 import (
-	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/util"
 	"gotest.tools/assert"
 	"testing"
 )
 
 func TestEnvFlag(t *testing.T) {
```

Comment:
it would be nice that the name of the function that is tested is also in the name of the name of the test function

Reply:
not part or this PR but yea of course you're right. I've changed it :)